tf-modules/cumulus/ecs_cluster_instance_autoscaling_cf_template.yml.tmpl

```diff
@@ -13,6 +13,7 @@ Resources:
         - DeviceName: "/dev/xvdcz"
           Ebs:
             DeleteOnTermination: true
+            Encrypted: true
```

Comment:
What's the impact to deployed clusters if ingest is ongoing?

Reply:
@Jkovarik The upgrade deployed to my stack fine. The existing EC2 instances are terminated, and new EC2 instances are created using the updated launch configuration that includes the encrypted drive. 

I imagine we should give guidance that any ingest in progress would fail while this upgrade is being applied.